Geometry/CaloTopology/interface/HGCalTBTopology.h

```diff
@@ -0,0 +1,132 @@
+#ifndef Geometry_CaloTopology_HGCalTBTopology_h
+#define Geometry_CaloTopology_HGCalTBTopology_h 1
+
+#include "DataFormats/ForwardDetId/interface/ForwardSubdetector.h"
+#include "DataFormats/DetId/interface/DetId.h"
+#include "Geometry/CaloTopology/interface/CaloSubdetectorTopology.h"
+#include "Geometry/HGCalCommonData/interface/HGCalGeometryMode.h"
+#include "Geometry/HGCalTBCommonData/interface/HGCalTBDDDConstants.h"
+#include <vector>
+#include <iostream>
+
+class HGCalTBTopology : public CaloSubdetectorTopology {
+public:
+  /// create a new Topology
+  HGCalTBTopology(const HGCalTBDDDConstants& hdcons, int subdet);
+
+  /// virtual destructor
+  ~HGCalTBTopology() override = default;
```

Comment:
Try removing this line and see if that helps the linking problem.

Reply:
That didn't help either